pkgs/development/python-modules/pyhomematic/default.nix

```diff
@@ -2,13 +2,13 @@
 
 buildPythonPackage rec {
   pname = "pyhomematic";
-  version = "0.1.71";
+  version = "0.1.72";
 
   disabled = !isPy3k;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0z09bw2mlhg7n9jyqmcyir306wpxr5nw1qsp5ps2iaw1qnyz5s9n";
+    sha256 = "sha256-odRBA7kEGNnIzeRpmhxnHVfRJGm+I6Rek7/ADfKO8Rs=";
   };
 
   # PyPI tarball does not include tests/ directory
```

Comment:
The linked issue has been closed. Can we get the tests up and running?

Reply:
We could set the delay to a high value, but who knows what would be high enough for Hydra, aarch64, ...

Reply:
If not please at least add a pythonImportsCheck.

Reply:
I enabled tests without changing the delay. I'll keep an eye out for build failures on Hydra.